### PR TITLE
feat(WFT-1030): Add whispir verify

### PR DESCRIPTION
**Pull request by Fatal-Errol**
Changes
- Add whispir verify endpoints to the postman collection

Output
image

### CODE REVIEW

File: Whispir_REST_API.postman_collection.json [hunk 1]
```diff
@@ -7388,6 +7388,1340 @@
...
+									"value": "Wed, 05 Jul 2023 14:02:28 GMT"
+								},
+								{
+									"key": "X-Cache",
+									"value": "Miss from cloudfront"
+								},
+								{
+									"key": "X-Amz-Cf-Id",
+									"value": "diqYjsEeyOT1GK1lzT9lszKl4YDrL1gI020nCl1lPa28hXlgHyGqWA=="
+								}
```
Comment (gabzi-whispir) [inline Whispir_REST_API.postman_collection.json]: Are we not supposed to mask the header values here, @Fatal-Errol ?

File: Whispir_REST_API.postman_collection.json [hunk 2]
```diff
@@ -7388,6 +7388,1340 @@
...
+							],
+							"cookie": [],
+							"body": "{\n    \"id\": \"azkP8DJEEsujuhwzO-O_N\",\n    \"channel\": \"sms\",\n    \"codeLength\": 6,\n    \"phoneNumber\": \"+63123456789\",\n    \"expiresAt\": \"2023-07-05T19:02:28.713Z\",\n    \"status\": \"pending\",\n    \"links\": {\n        \"verify\": \"https://api.au.whispir.com/otp/azkP8DJEEsujuhwzO-O_N/verify\"\n    }\n}"
```
Comment (gabzi-whispir) [inline Whispir_REST_API.postman_collection.json]: Is the `body.links.verify` value supposed to not be masked as well?

**Review by gabzi-whispir — CHANGES_REQUESTED** [review]
Pretty much the same sentiment for all the response values. I think we should at-least mask the possibly sensitive information in there, if not omit them altogether. Aside from these, great work! 💪

File: Whispir_REST_API.postman_collection.json (hunk 1 above)
Comment (Fatal-Errol) [reply Whispir_REST_API.postman_collection.json]: Hi @gabzi-whispir, I was instructed to remove this to make it consistent with the other endpoints.

File: Whispir_REST_API.postman_collection.json (hunk 2 above)
Comment (Fatal-Errol) [reply Whispir_REST_API.postman_collection.json]: this is a dummy value but I have removed it as per comment above

File: Whispir_REST_API.postman_collection.json (hunk 1 above)
Comment (whispir-sharie) [reply Whispir_REST_API.postman_collection.json]: That's right. Apologies if I didn't post my request here for visiblity. Thanks team 🙏

**Review by whispir-sharie — APPROVED** [review]
💯

**Pull request merged**